packages/nimble-components/src/table-column/extensions/fractional-width-column.ts

```diff
@@ -0,0 +1,49 @@
+import { attr, nullableNumberConverter } from '@microsoft/fast-element';
```

Comment:
I think we can call the directory mixins instead of extensions. I think it's a well-known enough term for JS users / we are using the typescript Mixins pattern (with some extra spice)